samples/bot-batch-operations/csharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:3978",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    ".NET Core": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:3979;http://localhost:3978",
```

Comment:
do we need this? `https://localhost:3979`

Reply:
please see the other samples, is an alternative port 
https://github.com/southworks/Microsoft-Teams-Samples/blob/main/samples/bot-initiate-thread-in-channel/csharp/Properties/launchSettings.json